app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx

```diff
@@ -2,12 +2,19 @@ import React, { useState } from "react";
 import { Modal } from "neetoui";
 import notesApi from "apis/notes";
 
-export default function DeleteAlert({ refetch, onClose, selectedNoteIds }) {
+export default function DeleteAlert({
```

Comment:
Please replace this custom component with neetoUI [Alert Component](https://neeto-ui.netlify.app/#/alert)